### ISSUE TITLE
[Bug] Itemstack clone is null on inventory close

### ISSUE THREAD

**Issue by Anonymous-cheese** [snowgears/shopbugs#231]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Error in console

### Environment

```markdown
- Server Version: Paper 55 1.17 Snapshot
- Shop Version: 1.8.2.7

config files are stock nothing changed.
```

### Server startup log

https://paste.gg/p/Toasthax/38bd8fc6c95d4c0d80f2881c2f5112be

With ERROR https://paste.gg/p/Toasthax/ea95b1d2e1d7470c86adb9a345db5444
### Shop config files

https://paste.gg/p/Toasthax/91887c46fbd64e67995fdf43dd9cc27e

### Bug description

Shops are completely broken can be opened by all users.

### Steps to reproduce

Happens on start up. 

### Expected behaviour

Working shops

### Actual behaviour

Shops are un-interactable.

**Comment by snowgears** [comment]
can you please post the full startup log? what you posted doesnt include Shop or the error.

**Comment by Anonymous-cheese** [comment]
> can you please post the full startup log? what you posted doesnt include Shop or the error.

https://paste.gg/p/Toasthax/ea95b1d2e1d7470c86adb9a345db5444

**Comment by snowgears** [comment]
Fixed in 1.8.2.8 (in progress)
